app/src/main/java/com/android/brewr/ui/overview/JourneyComponents.kt

```diff
@@ -131,61 +141,66 @@ fun CoffeeShopCheckRow(
             color = Color.Black,
             modifier = Modifier.testTag("coffeeShopCheckText"))
       }
+}
 
-  if (coffeeshopExpanded) {
```

Comment:
The way you updated this code is really clean, we will need update in this part as we need to allow users to input just name of the coffeeshop for the offline mode. I will first approve this pr and change the updated part in this style.